projects/scion/workbench/src/lib/content-projection/content-projection.directive.ts

```diff
@@ -58,9 +58,17 @@ export class ContentProjectionDirective implements OnInit, OnDestroy {
     this._contentViewRef = (await this.overlayHost).createEmbeddedView(this.contentTemplateRef, null);
     this._contentViewRef.detectChanges();
 
-    // (Re-)position content each time the bounding box element changes its size.
+    // Fixed content position.
+    this.styleContent({
+      position: 'fixed',
+    });
+
+    // (Re-)position visible content each time the bounding box element changes its size.
```

Comment:
I would write it as follows, also describing why we are ignoring the `0-dimension`.
```ts
    // (Re-)position content each time the bounding box element changes its size.
    fromDimension$(this._boundingBoxElement)
      .pipe(takeUntil(this._destroy$))
      .subscribe(dimension => {
        if (isNullDimension(dimension)) {
          // When removing the bounding box element (this directive's host) from the DOM, its dimension drops to 0.
          // We ignore this event to preserve the dimension of projected content, crucial, for example, if projected
          // content implements virtual scrolling. Otherwise, its content would reload when adding the host to the DOM again.
          // For example, inactive views are removed from the DOM.
          return;
        }
        this.stickContentToHostBoundingBox();
      });

function isNullDimension(dimension: Dimension): boolean {
  return dimension.offsetWidth === 0 && dimension.offsetHeight === 0;
}

```

Reply:
done